core/src/main/java/io/questdb/griffin/engine/table/SelectedRecord.java

```diff
@@ -145,6 +146,11 @@ public Long256 getLong256B(int col) {
         return base.getLong256B(getColumnIndex(col));
     }
 
+    @Override
+    public long getLongIPv4(int col) {
```

Comment:
This method is still not implemented in many places like `UnionRecord`, `UnionCastRecord`, all of the map records and so on. Considering that all of the existing implementations are the same, why don't we define it as the default impl in the `Record` interface?
```java
    default long getLongIPv4(int col) {
        return Numbers.ipv4ToLong(getIPv4(col));
    }
```